packages/flutter_markdown/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
-## NEXT
+## 0.6.15
 
+* Introduce a new `MarkdownElementBuilder.visitElementAfterWithContext()` method passing the widget `BuildContext` and
+  the parent texts' `TextStyle`.  
+  This should allow custom syntax implementations to get data from the current context, as well as properly style any
```

Comment:
The changelog should just describe the change (the first sentence); it doesn't need to provide the motivation for it.